rocAL/rocAL/include/pipeline/master_graph.h

```diff
@@ -95,7 +95,7 @@ class MasterGraph
     }
     void set_output(Image* output_image);
     bool empty() { return (remaining_count() < (_is_sequence_reader_output ? _sequence_batch_size : _user_batch_size)); }
-    size_t internal_batch_size() { return _internal_batch_size; }
+    size_t user_batch_size() { return _user_batch_size; }
```

Comment:
Do we need to introduce another api for getting user batch size?
is it not already available?

Reply:
IBS removal is added using my commit which added this function. Use this commit to remove this function:
https://github.com/SundarRajan28/MIVisionX/commit/6124b726db82b46139fdb138f8a499c66575ed11